rust/crates/sdk/Cargo.toml

```diff
@@ -11,7 +11,7 @@ exclude = ["/target/*", "examples/*", "tests/*", "docs/*", "book/*"]
 keywords = ["chainlink"]
 
 [dependencies]
-data-streams-report = { git = "https://github.com/smartcontractkit/data-streams-sdk.git", subdir = "rust/crates/report" }
+chainlink-data-streams-report = "0.1.0"
```

Comment:
do we know if publishing this crate will fail if the report SDK isn't published first?